medicinskij-academicheskij-jurnal.csl

```diff
@@ -0,0 +1,190 @@
+    <link href="http://www.zotero.org/styles/medicinskij-academicheskij-jurnal" rel="self"/>
+    <link href="http://protect.gost.ru/document.aspx?control=7&amp;id=173511" rel="documentation"/>
+    <author>
+      <name>Valery Mukhin</name>
+      <email>Valery.Mukhin@gmail.com</email>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="medicine"/>
+    <category field="biology"/>
+    <summary>This style is developed on the base of the gost-r-7-0-5-2008-numeric  by Yurkin Maxim;, and the main difference is that in this style the delimiters are dots with dashs. And all authors are included in bibliograohies. !!! IMPORTANT !!! Citations are numeric, order of bibliography is in order of citations, default locale is 'en'.    РЎС‚РёР»СЊ РѕСЃРЅРѕРІР°РЅ РЅР° СЃС‚РёР»Рµ &quot;gost-r-7-0-5-2008-numeric&quot;. РџРѕСЏСЃРЅРµРЅРёСЏ Рє РёСЃС…РѕРґРЅРѕРјСѓ СЃС‚РёР»СЋ СЃР»РµРґСѓСЋС‰РёРµ: РћСЃРЅРѕРІРЅР°СЏ РїСЂРѕР±Р»РµРјР° СЃРµР№С‡Р°СЃ - СЌС‚Рѕ Р»РѕРєР°Р»РёР·Р°С†РёСЏ С‚РµСЂРјРёРЅРѕРІ (РЎ., Рў., РїРѕРґ СЂРµРґ.,Рё РґСЂ. ... РёР»Рё СЃРѕРѕС‚РІРµС‚СЃС‚РІРµРЅРЅРѕ P., Vol., ed., et al.). Р’ РїСЂРёРЅС†РёРїРµ СЌС‚Рѕ СЂР°Р±РѕС‚Р°РµС‚ Рё РєРѕРЅС‚СЂРѕР»Р»РёСЂСѓРµС‚СЃСЏ РїРѕР»РµРј  default-locale (РІС‹С€Рµ, РІРѕ РІС‚РѕСЂРѕР№ СЃС‚СЂРѕС‡РєРµ). Р”Р»СЏ СЂСѓСЃСЃРєРѕРіРѕ РЅР°РґРѕ Р·Р°РјРµРЅРёС‚СЊ РЅР° &quot;ru-RU&quot;. Р­С‚Рѕ РѕРїСЂРµРґРµР»СЏРµС‚ РѕРґРЅРё РЅР°СЃС‚СЂРѕР№РєРё РґР»СЏ РІСЃРµР№ Р±РёР±Р»РёРѕРіСЂР°С„РёРё. РћРґРЅР°РєРѕ, РёРґРµР°Р»СЊРЅРѕРµ СЂРµС€РµРЅРёРµ - СЌС‚Рѕ РІС‹Р±РёСЂР°С‚СЊ СЏР·С‹Рє Р»РѕРєР°Р»РёР·Р°С†РёРё С‚РµСЂРјРёРЅРѕРІ РІ Р·Р°РІРёСЃРёРјРѕСЃС‚Рё РѕС‚ СЏР·С‹РєР° Р·Р°РїРёСЃРё (РєР°Рє Рё РїСЂРµРґРїРёСЃС‹РІР°РµС‚ Р“РћРЎРў). Р­С‚Рѕ РЅРµРІРѕР·РјРѕР¶РЅРѕ СЃРґРµР»Р°С‚СЊ РІ СЂР°РјРєР°С… Zotero 4.0 Рё CSL 1.0, РЅРѕ СѓР¶Рµ СЃРµР№С‡Р°СЃ С‡С‚Рѕ-С‚Рѕ РјРѕР¶РЅРѕ СЃРґРµР»Р°С‚СЊ РІ СЂР°РјРєР°С… MultiLingual Zotero - СЃРј. РЅР°РїСЂРёРјРµСЂ http://forums.zotero.org/discussion/18482/multilingual-zotero-fields-and-csl-variables.    Р РїРѕСЃР»РµРґРЅРµРµ, РїР°С‚РµРЅС‚С‹ РїРѕ СѓРјРѕР»С‡Р°РЅРёСЋ РїСЂРµРґСЃС‚Р°РІР»РµРЅС‹ РєР°Рє USA, РїРѕСЃРєРѕР»СЊРєСѓ РІ CSL 1.0 РЅРµС‚ РїРµСЂРµРјРµРЅРЅРѕР№, СЃРѕРѕС‚РІРµС‚СЃС‚РІСѓСЋС‰РµР№ РїРѕР»СЋ country РІ Р±Р°Р·Рµ Zotero 4.0. РќР°РґРµСЋСЃСЊ, СЌС‚Рѕ Р±СѓРґРµС‚ РёСЃРїСЂР°РІР»РµРЅРѕ РІ Р±СѓРґСѓС‰РёС… РІРµСЂСЃРёСЏС….</summary>
+    <published>2016-10-09T10:40:00+06:00</published>
+    <updated>2016-09-09T10:26:46+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale>
+    <terms>
+      <term name="issue" form="symbol">в„–</term>
+    </terms>
+  </locale>
+  <locale xml:lang="en">
+    <terms>
+      <term name="online">electronic resource</term>
+      <term name="letter" form="short">pat.</term>
+      <term name="editor" form="verb-short">ed.</term>
+      <term name="translator" form="verb-short">trans.</term>
+    </terms>
+  </locale>
+  <locale xml:lang="ru">
```

Comment:
Since you have a default-locale set to English, you can delete all russian terms